cg/meta/orders/status.py

```diff
@@ -145,13 +145,26 @@ def cases_to_status(cls, data: dict) -> dict:
         cases = cls.group_cases(data["samples"])
 
         for case_name, case_samples in cases.items():
-            priority = cls.get_single_value(case_name, case_samples, "priority", "standard")
+            cohorts = set(
+                cohort for sample in case_samples for cohort in sample.get("cohorts", set())
+            )
+            if "" in cohorts:
```

Comment:
Why this? It seems like `cohorts: Set[Set[str]]` (btw please add type hints), so why are we testing for empty string??

Reply:
since it was there and I didn't know how else to get rid of it

Reply:
fixed